src/runtime/storage/store-interface.ts

```diff
@@ -200,6 +194,17 @@ export abstract class ActiveMuxer<T extends CRDTTypeRecord> implements StorageCo
     }
   }
 
+  async idle(): Promise<void> {
+    return Promise.resolve();
+  }
+
+  async serializeContents(): Promise<T['data']> {
+    throw new Error('Method not implemented.');
+  }
+  async modelForSynchronization(): Promise<{}> {
+    return this.serializeContents();
+  }
+
   getStorageEndpoint() {
```

Comment:
this method can be removed as it is already in ActiveStore

Reply:
Ah I had made a mistake. The ActiveMuxer was implementing ActiveStore.. not extending it. I've made sure ActiveMuxer extends ActiveStore and now I can remove all these unnecessary methods.